apps/system/js/notifications.js

```diff
@@ -460,6 +464,17 @@ var NotificationScreen = {
     messageContent.setAttribute('dir', 'auto');
     message.appendChild(messageContent);
     notificationNode.appendChild(message);
+    var countDiv = document.createElement('div');
+    countDiv.classList.add('detail-count');
```

Comment:
So if I'm understanding this code right, only one notification per app id displayed, and it will contain the message of the earliest notifiction, and will then display a count of the number of notifications, but the messages in the subsequent notifications are just lost?  Is that really what UX wants?